tools/package-client.js

```diff
@@ -463,6 +566,38 @@ exports.publishPackage = function (options) {
     }
   }
 
+  // Check that our documentation exists (or we know that it doesn't) and has
+  // been filled out.
+  var readmeInfo;
+  buildmessage.enterJob("processing docs", function () {
+    readmeInfo = packageSource.processReadme();
+  });
+  if (buildmessage.jobHasMessages())
+    return;
+  if (readmeInfo && (readmeInfo.hash === files.blankHash)) {
+    buildmessage.error(
+      "Your documentation file is blank, so users may have trouble figuring " +
+      "out how to use your package. Please fill it out, or " +
+      "set 'documentation: null' in your Package.describe");
+  }
+
+  if (readmeInfo &&
+      readmeInfo.excerpt.length > 1500) {
```

Comment:
Wait, moving it to later in the code doesn't make any sense. I don't know why I was talking about it here.

I kind of like having a long and useful message pop up fairly early on, and there is precedent for doing this sort of pre-checking, so I am hesitant to remove it. Pre-checking is especially useful because if you run 'meteor publish --create' and get bounced *after* creating a version record, you now have to run 'meteor publish', which is kind of annoying. 

We could pre-flight metadata, but maybe in a different CL. I am not a fan of the extra back-and-forth with the server.

Though, if you tell me to remove this check, I will.